### ISSUE TITLE
Missing downward feature `runtime-tokio`.

### ISSUE THREAD

**Issue by xuxiaocheng0201** [SeaQL/sea-orm#2174]
## Description

Enabled `runtime-tokio` in `sea-orm` but panics
```
either the `runtime-async-std` or `runtime-tokio` feature must be enabled
```
in `sqlx-core`.

## Steps to Reproduce

1. Add `sea-orm = { version = "1.0.0-rc.3", features = ["sqlx-sqlite", "runtime-tokio"] }` to `Cargo.toml`
2. In `main.rs`:
```rust
use sea_orm::Database;

#[tokio::main]
async fn main() -> anyhow::Result<()> {
    let connection = Database::connect("sqlite://1.db?mode=rwc").await?;
    connection.ping().await?;
    Ok(())
}
```
3. Compile and run.

### Expected Behavior

Normally run.

### Actual Behavior

Panic:
```text
thread 'main' panicked at D:\AppData\Languages\Cargo\registry\src\index.crates.io-6f17d22bba15001f\sqlx-core-0.7.4\src\pool\inner.rs:52:24:
either the `runtime-async-std` or `runtime-tokio` feature must be enabled
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

### Reproduces How Often

Always.

### Workarounds

Missing downward feature.

## Reproducible Example

Codes above.

## Versions

I tested `1.0.0-rc.3` and `0.12.15`, both had this problem.

**Comment by billy1624** [comment]
Hey @xuxiaocheng0201, `runtime-tokio` is just a placeholder feature. It did nothing behind the scene. Please enable one of
- `runtime-tokio-native-tls`
- `runtime-tokio-rustls`

https://github.com/SeaQL/sea-orm/blob/0ff000b8f845dc469e98000410443cc008f60d8c/Cargo.toml#L116-L126

**Comment by xuxiaocheng0201** [comment]
> Hey @xuxiaocheng0201, `runtime-tokio` is just a placeholder feature. It did nothing behind the scene. Please enable one of
> 
> * `runtime-tokio-native-tls`
> * `runtime-tokio-rustls`

Thanks @billy1624 .

But I only use the `sqlite` backend, which needn't tls.

Enable `runtime-tokio-native-tls` or `runtime-tokio-rustls`
resulted in additional unnecessary dependencies.

**Comment by billy1624** [comment]
I see what you mean now. I just drafted a PR, would you like to try it? https://github.com/SeaQL/sea-orm/pull/2175

**Comment by tyt2y3** [comment]
Fix in #2171
